app/generator/api_combiner.rb

```diff
@@ -1,27 +1,57 @@
 module ApiCombiner
+  attr_reader :trail_species
+
   def self.get_species(trail_id)
-    gbif = GbifService.new
-    eol = EolService.new
     trail = Trail.find(trail_id)
-    trail_species = []
+    @trail_species = []
```

Comment:
Doesn't really look like you are doing anything with this instance variable & attr_reader, necessary?